tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -254,6 +255,69 @@ describe('metrics reducers', () => {
       );
     });
 
+    it('automatically pins cards from pre-pinned storage', () => {
```

Comment:
add a case where the unresolvedImportedPinnedCards are inconsistent with the cardMetadata. 

for instance, missing step/runId for image based tag.